### ISSUE TITLE
KIF does not seem to find back button.

### ISSUE THREAD

**Issue by matuszakpatryk** [kif-framework/KIF#1097]
I have a navbar with default back button. I have read that the accessibilityLabel of this back button should be equal to the title of the previous ViewController. My previous ViewController's title is **Login** so here is how I am trying to use that back button:
`tester().tapView(withAccessibilityLabel: "Login")`

Unfortunately, I am getting an error that KIF failed to find element with accessibility label == "Login"
I have tried even creating custom UIBarButtonItem and replace navigationItem.leftBarButtonItem but it does not work either.

Can someone help?
